swagger/api.yaml

```diff
@@ -28,13 +28,17 @@ paths:
   "/oauth2/v1/access_token":
     $ref: "./oauth.yaml#/endpoints/request_access_token"
 
-  # Catalog IDA
+  # Discovery IDA
+  # Base Discovery Endpoints
+  "/v1/courses":
+    $ref: "https://raw.githubusercontent.com/edx/course-discovery/e855990/api.yaml#/endpoints/v1/courses"
+  # Catalog Endpoints
   "/catalog/v1/catalogs":
-    $ref: "https://raw.githubusercontent.com/edx/course-discovery/649a4cb/api.yaml#/endpoints/v1/catalogs"
+    $ref: "https://raw.githubusercontent.com/edx/course-discovery/e855990/api.yaml#/endpoints/v1/catalogs"
```

Comment:
have you been able to confirm that we are not using this / not going to break any current usage?

Reply:
There doesn't seem to be any real way to test this not in prod, but this should be purely additive since the only difference in the commits is adding the courses endpoint